src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
```

Comment:
you should save color as a String

Reply:
then why would I need enum?

Reply:
I am using only Enum in every method and there is nothing about that in readme file, only create Ball class with fields color and number.
[README.md](https://github.com/mate-academy/jv-lottery/files/7896420/README.md)

Reply:
<img width="855" alt="Screenshot 2022-01-19 at 13 08 27" src="https://user-images.githubusercontent.com/33026471/150118570-84c7f62b-a65f-4957-bd7b-602588ab304a.png">

getRandomColor method must return String as written in the task description. Now, if we receive random color in String format it's easier to save it as a String, than converting to some other type again. And enum is a good way to store some constant values.

Reply:
> <img alt="Screenshot 2022-01-19 at 13 08 27" width="855" src="https://user-images.githubusercontent.com/33026471/150118570-84c7f62b-a65f-4957-bd7b-602588ab304a.png">
> 
> getRandomColor method must return String as written in the task description. Now, if we receive random color in String format it's easier to save it as a String, than converting to some other type again. And enum is a good way to store some constant values.

oops, didn`t notice that, only looked at this: 

> create Ball class with the following fields: color and number and override toString() method

and there was nothing about type of that variables, so I was choosing myself.
last commit fixes everything.